src/llmq/utils.cpp

```diff
@@ -954,6 +954,11 @@ bool IsQuorumTypeEnabledInternal(Consensus::LLMQType llmqType, const CQuorumMana
             }
             break;
         }
+        case Consensus::LLMQType::LLMQ_25_67:
+            if (!IsV19Active(pindex)) {
```

Comment:
This changes test conditions from having active platform quorums _prior to v19 hf_ (mainnet/testnet) to activating them _after v19 hf_ (testnet only). There is a risk of missing potential issues which might occur during the migration of platform quorums from regular mns to hpmns (which would be skipped on testnet with this change). And this could result in having a "surprise" on mainnet later. I think we need to activate llmq_25_57 earlier and let it run at least for a day before activating v19 on testnet. Could use a simple "hf at a specific blockheight" activation here maybe i.e.

```suggestion
            if (pindex->nHeight < SOME_HEIGHT) {
```

Reply:
@UdjinM6 hmm I see. Any idea of the value of `SOME_HEIGHT` ?

Reply:
Testnet is at ~845400 now. Could add 2-3 days (we need to build it and to deploy it with no rush and without the need for `invalidateblock` etc. if we are too late). Could be block 847000 (assuming ~576 blocks per day) or somewhere around it.